index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```

Comment:
- [OPTIONAL] This requested change is optional as your first reviewer didn't mention it, Kindly, try to wrap this `h1` tag inside a `header` tag for more semantic reasons ^^